Phone and tablet UX + mmcboot support (#67)
Tow-Boot/Tow-Boot: boards/pine64-pinephoneA64/README.md

# PINE64 Pinephone (A64)

## Device-specific notes

There is no display output for this device.

## Additional features

The phone can be started in *USB Mass Storage* mode by holding the *volume up*
button at startup before and during the second vibration. The LED will turn
blue if done successfully. In this mode, the phone will work like a USB drive
when connected to a host computer.

> Booting an operating system, or an installer written to an SD card will require
> holding the *volume down* button before and during the second vibration. When
> done successfully, the LED will turn aqua.

Reviewer: ```suggestion
Booting from an SD card will require holding the *volume down* button before and
during the second vibration. When done successfully, the LED will turn aqua.
```
The way you had it worded could be interpreted as saying "Booting an operating system will require holding down the volume down button".

Author: I edited the wording to include *operating system* still, as I try very hard in the documentation to strongly delineate between *platform firmware startup* and *operating system boot*. This is because the former is generally static and set in hardware, while the latter is handled by the platform firmware, and can be anything.
